osbuild/treesum.py

```diff
@@ -0,0 +1,58 @@
+import errno
+import json
+import os
+
+def treesum(m, dir_fd):
```

Comment:
Missing `__all__`, which you added in all other modules. Not a blocker for merging, of course.

Reply:
Ah, thanks, that was an oversight. I'll add that in a follow-up.